src/node_crypto.cc

```diff
@@ -1430,10 +1430,13 @@ int SSLWrap<Base>::NewSessionCallback(SSL* s, SSL_SESSION* sess) {
   memset(serialized, 0, size);
   i2d_SSL_SESSION(sess, &serialized);
 
+  unsigned int session_id_length;
+  const unsigned char *session_id = SSL_SESSION_get_id(sess,
```

Comment:
`char*` (star leans left)

Reply:
Done.

Reply:
@bnoordhuis is there a way to solve these issues with a new linter? On JS side there are almost no style nits since quite a while but I have seen a lot of those for C++. A issue like this one should be not to difficult to spot for a linter, right?

I do not write C++ normally, so I do not know if there is a good alternative to cpplint but google gave me e.g. http://clang.llvm.org/extra/clang-tidy/ and it seems to be extendable. So we could also add more rules if we are not happy with the PRs.

Reply:
`clang-format` is wonderful and quite configurable. There's even a mode where it only applies to the code you touched in your diff. A lot of parts of Chromium require that running through that to upload a change.

Reply:
Apropos `clang-format`, see the discussion in #3612.

Reply:
💯 on having a C++ linter, looking at #3612 it sounds like the massive diff was an issue, but most people were +1 on the idea of having linting.

Given how much we use `eslint`, and that we've had more and more sweeping changes from it, I think we're probably more open to these kinds of refactorings these days. It could even just start with one rule (like `star leans left`). Auto fixing issues sounds good too.

Reply:
Not just the diff was a problem, clang is hard to get, linux distro packaged versions are out of date, and each version of the formatter seems to do things differently. I think packaging for OS X and Windows was even more problematic than linux. If it was stable, easy to get, widely available, it'd would have flown further.

Reply:
Maybe packaging it as an npm module might be helpful then.